sdk/ml/azure-ai-ml/azure/ai/ml/entities/_load_functions.py

```diff
@@ -375,6 +363,15 @@ def load_component(
 
     :return: A function that can be called with parameters to get a `azure.ai.ml.entities.Component`
     :rtype: Union[CommandComponent, ParallelComponent, PipelineComponent]
+
+    .. admonition:: Example:
+        :class: tip
+        .. literalinclude:: ../../../../samples/ml_samples_component_configurations.py
+            :start-after: [START configure_load_component]
+            :end-before: [END configure_load_component]
+            :language: python
+            :dedent: 8
+            :caption: Configuring load component.
```

Comment:
nit: I think "Loading a Component object from a YAML file" would be a clearer caption.

Reply:
ok